translate/assign/py_assign.py

```diff
@@ -11,6 +11,7 @@
 def assign(node):
     gast = {}
     gast["type"] = "varAssign"
+    gast["kind"] = "let"
```

Comment:
Is this the way non-constant variables are being annotated in the gAST? Seems pretty js-specific.

Reply:
That's the way we decided to do it across all languages to make things easier but agreed it does seem a bit "hard-coded"